### PR TITLE
Update INSTALL

### PR DESCRIPTION
**Pull request by eduardomozart**
Updated some Cygwin dependencies that were deprecated.

### CODE REVIEW

**Comment by TheWitness** [comment]
Thanks @eduardomozart

**Review by TheWitness — APPROVED** [review]

**Review by TheWitness — APPROVED** [review]

File: INSTALL [hunk 1]
```diff
@@ -147,4 +147,4 @@ Known Issues
 
 
 -----------------------------------------------
-Copyright (c) 2004-2023 - The Cacti Group, Inc.
\ No newline at end of file
+Copyright (c) 2004-2023 - The Cacti Group, Inc.
```
Comment (cigamit) [inline INSTALL]: Shouldn't this be 2024?

**Review by cigamit — APPROVED** [review]

File: INSTALL (hunk 1 above)
Comment (TheWitness) [reply INSTALL]: Fixed.

**Comment by TheWitness** [comment]
Thanks @cigamit

**Pull request merged**